### PR TITLE
[test] DA-4538 - Skip disabled projects in APIs

### PR DESCRIPTION
**Pull request by linonymous**
* DA-4538
- Skip disabled projects in APIs

Signed-off-by: linonymous <linonymous@gmail.com>
(cherry picked from commit 009955187a1a181e2b6dc3057cdac66fea04bb87)

### CODE REVIEW

File: affiliation/service.go [hunk 1]
```diff
@@ -123,6 +123,8 @@ type Service interface {
 	isPrecacheRunning() bool
 	setPrecacheRunning()
 	clearPrecacheRunning()
+	IsProjectSkipped(string) bool
+	SkipDisabledProjects(project string) string
```
Comment (ajinkyan83) [inline affiliation/service.go:127]: nit: better to use a `pointer receiver` here.

Comment (linonymous) [reply affiliation/service.go:127]: this string is going to be small, & perhaps, changing it to pointer would make it mutable, moreover, still we can change it, but the callers of these functions are too many to update for this trivial change. & this particular change is already merged to master branch, check it here: https://github.com/LF-Engineering/dev-analytics-affiliation/pull/123

Comment (linonymous) [reply affiliation/service.go:127]: we can revisit this, when we update this again.

**Review by ajinkyan83 — APPROVED** [review]
/lgtm

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Comment by lukaszgryglicki** [comment]
We will need the same PR for `prod` branch as well.
cc @linonymous

**Pull request merged**